.github/workflows/swift-toolchain.yml

```diff
@@ -325,13 +330,15 @@ jobs:
 
       - uses: compnerd/gha-setup-vsdevenv@main
         with:
-          host_arch: amd64
+          host_arch: ${{ inputs.build_arch }}
           components: 'Microsoft.VisualStudio.Component.VC.Tools.x86.x64;Microsoft.VisualStudio.Component.VC.Tools.ARM64'
           arch: ${{ matrix.arch }}
 
+      - uses: seanmiddleditch/gha-setup-ninja@master
+        if: inputs.build_os == 'Darwin'
```

Comment:
If we are doing this, can we perhaps file an issue for a follow up which uses setup-cmake and gha-setup-ninja to configure those rather than the vsdevenv on Windows?

Reply:
Filed https://github.com/compnerd/swift-build/issues/821